### PR TITLE
4.3.0

### PR DESCRIPTION
**Pull request by xHyroM**
Install this RP:
```ssh
npm i Garlic-Team/GCommands#dev
```

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - name: Create Folder
+      run: mkdir docs
+
+    - name: Download
+      run: npm i jsdoc-to-markdown
+
+    - name: Build
+      run: npm run docs:build
+
+    - name: copy file
```
Comment (xHyroM) [inline .github/workflows/docs.yml:24]:
```suggestion
    - name: Copy File
```

File: typings/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    public setMinValues(min: number): this;
+    public setID(id: number): this;
+    public addOption(option: MessageSelectMenuOption)
+    public addOptions(...options: MessageSelectMenuOption[])
+    public removeOptions(index: number, deleteCount: number, ...options: MessageSelectMenuOption[])
+    public toJSON(): this;
+  }
+
+  export class MessageSelectMenuOption {
+    constructor(data: MessageButton)
```
Comment (xHyroM) [inline typings/index.d.ts:229]:
```suggestion
    constructor(data: MessageSelectMenu)
```

**Review by Gabe616 — APPROVED** [review]
noice

**Review by iZboxo — APPROVED** [review]

**Review by SebestikCZ — APPROVED** [review]
Ok, I think good.

**Comment by xHyroM** [comment]
Okay, I go make final test

**Comment by xHyroM** [comment]
[EmbedBuilder](https://github.com/Garlic-Team/Tutorials/blob/main/features/commands/embedbuilder.js)

**Review by Gabe616 — APPROVED** [review]
why'd they remove structures doe

**Comment by xHyroM** [comment]
> why'd they remove structures doe

https://github.com/discordjs/discord.js/issues/6039#issuecomment-873621116

**Review by iZboxo — APPROVED** [review]

**Review by iZboxo — COMMENTED** [review]
Like it

**Pull request merged**